thetis/shallowwater_eq.py

```diff
@@ -122,6 +122,39 @@
     adv_term = shallowwater_eq.HorizontalAdvectionTerm(...)
     adv_form = adv_term.residual(..., bnd_conditions=swe_bnd_funcs)
 
+------------------
+Wetting and drying
+------------------
+
+If the option :attr:`.ModelOptions.wetting_and_drying` is ``True``, then wetting and drying is included through the formulation of Karna et al. (2010).
+
+The method introduces a modified bathymetry :math:`\tilde{h} = h + f(H)`, which ensures positive total water depth, with :math:`f(H)` defined by
+
+.. math::
+   f(H) = \frac{1}{2}(\sqrt{H^2 + \alpha^2} - H),
+
+introducing a wetting-drying parameter :math:`\alpha`, with dimensions of length. This results in a modified total water depth :math:`\tilde{H}=H+f(H)`.
+
+The value for :math:`\alpha` is specified by the user through the option :attr:`.ModelOptions.wd_alpha`, in units of metres. The default value for :attr:`.ModelOptions.wd_alpha` is 0.5, but the appropriate value is problem specific and should be set by the user.
```

Comment:
metres -> meters. Options say "Unit is meters". Either spelling is fine as long as we're consistent.